.github/workflows/actions/gradle/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Gradle command action'
+inputs:
```

Comment:
Guessing this was here for testing since the workflows are using the common one now